### ISSUE TITLE
Can not work with shopify theme sections 

### ISSUE THREAD

**Issue by raihan004** [Shopify/shopify_theme#226]
Shopify recently update there theme structure.
But theme gem does not recognise new folder sections.
I edited theme config file
:whitelist_files:
- layout/
- assets/
- config/
- snippets/
- templates/
- locales/
- sections/

but it does not work.

> 'sections/article-template.liquid' is not in a valid file for theme uploads Files
>  need to be in one of the following subdirectories: layout/ assets/ config/ snippets/ templates/ locales/

How can I solve this problem?

**Comment by fredrikekelund** [comment]
See #216.

Looks like development has gone a bit stale with this project, no code updates for over a year, and not much activity in the issue tracker either. Kind of a shame seeing as this is such a key module of building Shopify themes and they seem to take pride in their open source participation...

**Comment by Oscarteg** [comment]
I think this is the new command line tool https://github.com/Shopify/themekit

**Comment by ckhatton** [comment]
Of course! There I was waiting for a response on a issue I posted here (facepalm)
